### PR TITLE
Feat/3825  user activity feed

### PR DESCRIPTION
**Pull request by asf-algoo**
See #3825.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by raphj — APPROVED** [review]

File: frontend/src/container/Sidebar.jsx [hunk 1]
```diff
@@ -90,6 +91,19 @@ export class Sidebar extends React.Component {
     )
   }
 
+  createLink = (label, to) => {
```
Comment (gbonaspetti) [inline frontend/src/container/Sidebar.jsx:94]: Just a suggestion of a more explicit name, if you prefer `createLink` you can resolve this conversation
```suggestion
  getSidebarItem = (label, to) => {
```

Comment (grignards) [reply frontend/src/container/Sidebar.jsx]: Changed.

**Comment by gbonaspetti** [comment]
After testing:
- The "Activity feed" is not active in sidebar
Screenshot from 2020-11-25 14-01-50

- There is a icon tooltip with its name
Screenshot from 2020-11-25 14-05-04

- Missing a space between the Refresh button and the title zone
Screenshot from 2020-11-25 14-04-39

**Comment by gbonaspetti** [comment]
Tested on branch.

**Pull request merged**